pkg/controller/storagecluster/noobaa_system_reconciler.go

```diff
@@ -87,6 +92,11 @@ func (r *ReconcileStorageCluster) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *ocs
 	nb.Labels = map[string]string{
 		"app": "noobaa",
 	}
+	// If independent mode, we pass the endpoint value as a label to Noobaa
+	// so it can connect to the RGW S3 endpoint
+	if sc.Spec.ExternalStorage.Enable {
+		nb.Labels[externalRgwEndpointLabelName] = externalRgwEndpoint
```

Comment:
Instead of saving the variable between functions, could you instead Get() the RGW StorageClass, verify it exists, and read the value from there? Then we could keep the consts in one file and avoid the global var.

Reply:
This may also make it easier to refactor in case we ever want to use this for converged use cases, like in baremetal.

Reply:
IMHO, it feels overkill to fetch the SC, this var is initialized at the same time we build the parameters for the SC.

The converged mode will never have such use case, this is a small patch for 4.5 only. 4.6 will change and have a similar experience for both converged and independent.

Reply:
I also dislike one-off fixes that are supposed to be reverted later, but  as long as the whole process will be overhauled I'll live with it. :)